canonical/ubuntu-com-security-api
# Create numeric cves

## Done

- Added a new column `numerical_ids` to the `cves` table 
- Added a flask cli command to update `numerical_ids` for each cve
- Added `cves` field to `/security/notices`
  - i.e `/security/notices?cves=<cve-id>,<cve-id2>` 
- Updated `details` field by removing `cve_ids` from details search

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- After running the site locally, and migrations pass, update the new column with values using
```bash
flask insert_numerical_cve_ids
```
- View the site locally in your web browser at: http://0.0.0.0:8030/
- Run through the following [QA steps](https://canonical-web-and-design.github
.io/practices/workflow/qa-steps.html)
- Make a request to `/security/notices?cves=<cve-id>,<cve-id2>` and confirm that the result
has notices that include the specified `cve-ids`


## Issue / Card

Fixes [WD-13138](https://warthogs.atlassian.net/browse/WD-13138?atlOrigin=eyJpIjoiMWEyMjc3MDU0OGFiNGI3NzgzZjliM2I1YzcyMTQyZGQiLCJwIjoiaiJ9)


[WD-13138]: https://warthogs.atlassian.net/browse/WD-13138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
